subsys/usb/usb_device.c

```diff
@@ -172,7 +172,9 @@ static struct usb_dev_priv {
 static void usb_print_setup(struct usb_setup_packet *setup)
 {
 	/* avoid compiler warning if SYS_LOG_DBG is not defined */
-	setup = setup;
+#ifndef CONFIG_SYS_LOG
+	ARG_UNUSED(setup);
+#endif
```

Comment:
FWIW: the ARG_UNUSED() macro is a noop if the argument **is** used, so there's no need to put it inside an #ifdef.